tst/EnergyPlus/unit/DataSurfaces.unit.cc

```diff
@@ -244,6 +244,154 @@ TEST_F(EnergyPlusFixture, SurfaceTest_Surface2D)
     }
 }
 
+TEST_F(EnergyPlusFixture, SurfaceTest_Surface2D_bigVertices)
```

Comment:
2 new unit tests added related to the failed assert in `Surface2D`. The first is a convex surface, the second is nonconvex. In hindsight, this isn't focusing on the root problems.

Reply:
Do you feel more tests are justified?  Or is this still covering the issue _enough_?